diff -Naur --exclude '*~' Imaging-1.1.6/setup.py Imaging-1.1.6-lib64/setup.py
--- Imaging-1.1.6/setup.py	2006-12-03 12:37:29.000000000 +0100
+++ Imaging-1.1.6-lib64/setup.py	2008-07-08 14:28:23.000000000 +0200
@@ -16,7 +16,7 @@
 
 def libinclude(root):
     # map root to (root/lib, root/include)
-    return os.path.join(root, "lib"), os.path.join(root, "include")
+    return os.path.join(root, sys.lib), os.path.join(root, "include")
 
 # --------------------------------------------------------------------
 # Library pointers.
@@ -140,12 +140,9 @@
             add_directory(library_dirs, "/opt/local/lib")
             add_directory(include_dirs, "/opt/local/include")
 
-        add_directory(library_dirs, "/usr/local/lib")
-        # FIXME: check /opt/stuff directories here?
-
         prefix = sysconfig.get_config_var("prefix")
         if prefix:
-            add_directory(library_dirs, os.path.join(prefix, "lib"))
+            add_directory(library_dirs, os.path.join(prefix, sys.lib))
             add_directory(include_dirs, os.path.join(prefix, "include"))
 
         #
